src/main/java/com/redhat/lightblue/healthcheck/model/HealthCheckStatus.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.lightblue.healthcheck.model;
+
+public class HealthCheckStatus {
+
+    public enum Status {
+        success("{\"status\":\"success\"}"),
+        error("{\"status\":\"error\",\"message\":\"[healthcheckstatus.message]\"}");
```

Comment:
Changed this around significantly, and it seems much simpler now (to me at least)